Library/Categories/CoreData/NSManagedObject/NSManagedObject+MagicalDataImport.m

```diff
@@ -72,12 +72,14 @@ - (BOOL)MR_shouldImportData:(id)relatedObjectData forRelationshipNamed:(NSString
 
 #pragma mark - Setting Attributes and Relationships
 
-- (void)MR_setObject:(NSManagedObject *)relatedObject forRelationship:(NSRelationshipDescription *)relationshipInfo
-{
+- (void)MR_setObject:(NSManagedObject *)relatedObject forRelationship:(NSRelationshipDescription *)relationshipInfo{
+#pragma GCC diagnostic push
+#pragma GCC diagnostic ignored "-Wunused-variable"
```

Comment:
You are absolutely right! These should all be clang! That was an older codeSnippet.